include/uv/unix.h

```diff
@@ -509,4 +509,6 @@ typedef struct {
 #define UV_FS_O_SEQUENTIAL    0
 #define UV_FS_O_TEMPORARY     0
 
+#define UV_SPAWN_BUFFER_SIZE (64 * 1024)
```

Comment:
Don't define it in uv.h, that's a public header. I also don't think it really needs a define, just use the number directly.